barretenberg/cpp/src/barretenberg/plonk_honk_shared/composer/permutation_lib.hpp

```diff
@@ -27,15 +27,15 @@
 namespace bb {
 
 /**
- * @brief cycle_node represents the index of a value of the circuit.
+ * @brief cycle_node represents the idx of a value of the circuit.
  * It will belong to a CyclicPermutation, such that all nodes in a CyclicPermutation
  * must have the value.
  * The total number of constraints is always <2^32 since that is the type used to represent variables, so we can save
  * space by using a type smaller than size_t.
  */
 struct cycle_node {
-    uint32_t wire_index;
-    uint32_t gate_index;
+    uint32_t wire_idx;
```

Comment:
This file was using both `idx` and `index` arbitrarily so I updated to use only `idx` everywhere